libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -152,11 +152,25 @@ bool GCS_MAVLINK::init(uint8_t instance)
         return false;
     }
 
+    // PARAMETER_CONVERSION - Added: May-2025 for ArduPilot-4.7
+    // convert parameters; we used to use bits in the UARTDriver to
+    // remember whether the mavlink connection on that interface was
+    // "private" or not, and whether to ignore streamrate sets GCSs.
```

Comment:
Again - not sure if this is just stream rate sets from a GCS only.

Reply:
Reworded this too - thanks!